src/frontend/css/plugins/color.js

```diff
@@ -0,0 +1,64 @@
+const colorFunctions = require('less/lib/less/functions/color').default;
+
+const SUPPORTED_COLOR_FUNCTIONS = {
+  LIGHTEN: 'lighten',
+  DARKEN: 'darken',
+  FADE: 'fade'
+};
+
+// TODO (esn-frontend-common-libs#51): Write tests for this plugin
+function colorFunctionInterceptor(color, amount, method, functionName) {
+  if (!Object.values(SUPPORTED_COLOR_FUNCTIONS).includes(functionName)) throw new Error(`The LESS's color function '${functionName}' is not supported with CSS Variables`);
```

Comment:
Again, not sure if our frontend code support `.includes`

Reply:
If we leave tests aside, arrow functions and `Array.prototype.includes` can be used in all the browsers that OpenPaaS needs to support:

- https://caniuse.com/#search=arrow%20functions
- https://caniuse.com/#search=includes
- List of OpenPaaS' supported browsers: https://github.com/linagora/openpaas-esn/wiki/MVP-environment#browsers

However, I'm not too sure how our tests will be run. If we use a new headless browser or we transpile all our code (using Babel) before testing, there should be no problem. We'll need to ask @MichaelBailly about what's going to be done about frontend tests.

Reply:
We will transpile